<launch>
  <arg name="robot_name" default="sapienzbot" />

  <node pkg="rockin_logger" name="rockin_logger" type="rockin_logger" output="screen"> 
    <param name="pcd_file" value="$(find rockin_logger)/data/example.pcd" /> 
    <param name="image_file" value="$(find rockin_logger)/data/example.png" />
    <param name="robot_name" value="$(arg robot_name)" />
  </node>

  <!--node name="audio_capture" pkg="audio_capture" type="audio_capture" output="screen">
    <param name="bitrate" value="128"/>
    <remap from="audio" to="/rockin/audio"/>
  </node-->
  
  <node pkg="tf" type="static_transform_publisher" name="base_static_transform" args="0 0 0 0 0 0 /$(arg robot_name)/base_frame /base_frame 10" />
  <node pkg="tf" type="static_transform_publisher" name="laser_static_transform" args="0 0 0 0 0 0 /$(arg robot_name)/laser_frame /laser_frame 10" />
  <node pkg="tf" type="static_transform_publisher" name="camera_static_transform" args="0 0 1 0 0 0 /base_frame /rgb_frame 10" />
  <node pkg="tf" type="static_transform_publisher" name="depth_static_transform" args="0.1 0 1 0 0 0 /base_frame /depth_frame 10" />
    
    
</launch>

